client/src/pages/automation/project/utils/getNestedObject.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const getNestedObject = (jsonObj: any, selector: string) => {
+    const selectors = selector.split('.');
+
+    return selectors.reduce((obj, key) => {
```

Comment:
`obj` preimenuj u `selector`

Reply:
@kresimir-coko to nije selector, to je objekt nad kojim se vrši selektoranje, a slector je argument same metode